docs/src/pages/[platform]/components/radiogroupfield/examples/ThemeExample.tsx

```diff
@@ -6,28 +6,23 @@ import {
 } from '@aws-amplify/ui-react';
 
 const theme: Theme = {
-  name: 'radio-theme',
+  name: 'radiogroup-theme',
   tokens: {
-    colors: {
-      border: {
-        // Radio focus state inherits this global focus color
-        focus: { value: '{colors.brand.secondary.80}' },
-      },
-      font: {
-        primary: { value: '{colors.brand.secondary.60}' },
-        secondary: { value: '{colors.brand.secondary.40}' },
-      },
-    },
     components: {
-      radio: {
+      radiogroup: {
+        label: {
+          color: { value: '{colors.brand.secondary.40}' },
+        },
```

Comment:
May want to check the contrast ratio?

Reply:
thanks I've updated the colors to increase the contrast in light mode